### PR TITLE
update checkbox as empty

### PR DESCRIPTION
**Pull request by giwa**
fix the case where set null if checkbox is empty 

https://gyazo.com/be6f9790ed53b6e1b85b6b8b2d386641

### CODE REVIEW

File: src/main/java/org/embulk/output/kintone/KintoneColumnVisitor.java [hunk 1]
```diff
@@ -105,9 +104,15 @@ private void setTimestampValue(String fieldCode, Instant instant, ZoneId zoneId,
...
+
+        if (str != null && !str.equals("")){
+            List<String> values = Arrays.asList(str.split(valueSeparator, 0));
+            checkBoxFieldValue = new CheckBoxFieldValue(values);
+        }
         record.putField(
-            fieldCode,
-            new CheckBoxFieldValue(str.split(valueSeparator, 0))
+                fieldCode,
+                checkBoxFieldValue
```
Comment (abetomo) [inline src/main/java/org/embulk/output/kintone/KintoneColumnVisitor.java]: Are the indentations too deep?

Comment (giwa) [reply src/main/java/org/embulk/output/kintone/KintoneColumnVisitor.java]: Intellj aligned code. I fixed this.

Comment (giwa) [reply src/main/java/org/embulk/output/kintone/KintoneColumnVisitor.java]: Understand. I follow the IntelliJ auto line breaking.

**Review by abetomo — APPROVED** [review]
👍

**Review by d-hrs — APPROVED** [review]
lgtm

**Pull request merged**